gcp_pilot/exceptions.py

```diff
@@ -55,3 +55,6 @@ class PushWebhookInvalid(ValidationError):
 
 class ChannelIdNotUnique(Exception):
     pass
+
+class NotACalendarUser(Exception):
```

Comment:
This seems to be a typo of not authenticated issue (just by reading the logs you attached).
Maybe we can inherit that from `exceptions.NotAllowed` to avoid creating a bunch of very very specific exceptions, making it harder for client users to capture all of them.